sdk/translation/Azure.AI.Translation.Text/src/Custom/TextTranslationTranslateOptions.cs

```diff
@@ -0,0 +1,200 @@
+        /// </summary>
+        public ProfanityMarker? ProfanityMarker { get; set; }
+        /// <summary>
+        /// Specifies whether to include alignment projection from source text to translated text.
+        /// Possible values are: true or false (default).
+        /// </summary>
+        public bool? IncludeAlignment { get; set; }
+        /// <summary>
+        /// Specifies whether to include sentence boundaries for the input text and the translated text.
+        /// Possible values are: true or false (default).
+        /// </summary>
+        public bool? IncludeSentenceLength { get; set; }
+        /// <summary>
+        /// Specifies a fallback language if the language of the input text can&apos;t be identified.
+        /// Language autodetection is applied when the from parameter is omitted. If detection fails,
+        /// the SuggestedFrom language will be assumed.
+        /// </summary>
+        public string SuggestedFrom { get; set; }
+        /// <summary>
+        /// Specifies the script of the input text.
+        /// </summary>
+        public string FromScript { get; set; }
+        /// <summary>
+        /// Specifies the script of the translated text.
+        /// </summary>
+        public string ToScript { get; set; }
+        /// <summary>
+        /// Specifies that the service is allowed to fall back to a general system when a custom system doesn&apos;t exist.
+        /// Possible values are: true (default) or false.
+        ///
+        /// AllowFallback=false specifies that the translation should only use systems trained for the category specified
+        /// by the request. If a translation for language X to language Y requires chaining through a pivot language E,
+        /// then all the systems in the chain (X → E and E → Y) will need to be custom and have the same category.
+        /// If no system is found with the specific category, the request will return a 400 status code. AllowFallback=true
+        /// specifies that the service is allowed to fall back to a general system when a custom system doesn&apos;t exist.
+        /// </summary>
+        public bool? AllowFallback { get; set; }
+
+        /// <summary> Initializes new instance of TextTranslationTranslateOptions. </summary>
+        /// <param name="targetLanguages">
+        /// Specifies the language of the output text. The target language must be one of the supported languages included
+        /// in the translation scope. For example, use to=de to translate to German.
+        /// It&apos;s possible to translate to multiple languages simultaneously by repeating the parameter in the query string.
+        /// For example, use to=de&amp;to=it to translate to German and Italian.
+        /// </param>
+        /// <param name="content"> Array of the text to be translated.</param>
+        public TextTranslationTranslateOptions(IEnumerable<string> targetLanguages, IEnumerable<string> content)
```

Comment:
Have you considered to check for the empty `IEnumerable`? This would result into the 400 response. If you will add the check, please add it to all ctors where the `IEnumerable` is used.

Reply:
Yes.  The following code in the constructor adds the requested checks:
            
            Argument.AssertNotNull(targetLanguages, nameof(targetLanguages));
            Argument.AssertNotNull(content, nameof(content));